connectors/meilisearch/index.js

```diff
@@ -380,14 +380,13 @@ module.exports = async ({ storeConnector, collectionConnector }) => {
     /**
      * Get list of index uids in Meilisearch instance.
      *
-     * @returns {number[]} - Index uids
+     * @returns {Promise<number[]>} - Index uids
      */
     getCollectionsIndexedInMeiliSearch: async function (collections) {
-      const client = MeiliSearch({ apiKey, host })
-      let indexes = await client.getIndexes()
-      indexes = indexes.map(index => index.uid)
+      const indexes = await this.getIndexes()
+      const indexUids = indexes.map(index => index.uid)
```

Comment:
I know it seems too much, but this blank lines help a lot to understand the code :)
```suggestion
      const indexUids = indexes.map(index => index.uid)

```